deployment/ccip/changeset/cs_add_chain.go

```diff
@@ -110,48 +135,111 @@ func NewChainInboundChangeset(
 	}, nil
 }
 
+type AddDonAndSetCandidateChangesetConfig struct {
+	HomeChainSelector uint64
+	FeedChainSelector uint64
+	NewChainSelector  uint64
+	PluginType        types.PluginType
+	TokenConfig       TokenConfig
```

Comment:
I don't think this should be needed TokenConfig is part of the plugin config?

Reply:
Removed